### PR TITLE
Add pubkey to BcpConnection.getAccount

### LINKED ISSUE #436: Let BcpConnection.getAccount accept a pubkey
The interface `getAccount: (account: BcpAccountQuery) => Promise<BcpQueryEnvelope<BcpAccount>>;` remains unchanged but `BcpAccountQuery` gets an additional type, like:

```ts
declare type BcpAccountQuery = BcpAddressQuery | BcpPubkeyQuery | BcpValueNameQuery;
```

`BcpPubkeyQuery` must be created, see `BcpAddressQuery`. Pubkey should be of type `PublicKeyBundle`.

BNS, Lisk and Rise should support query by pubkey too (either by conversion to address or by querying by pubkey directly)

### PR DESCRIPTION
**Pull request by LucasIsasmendi**
Closes #436

### CODE REVIEW

File: packages/iov-bcp-types/src/bcp.ts [hunk 1]
```diff
@@ -91,13 +91,25 @@ export interface BcpValueNameQuery {
   readonly name: string;
 }
 
-export type BcpAccountQuery = BcpAddressQuery | BcpValueNameQuery;
+export interface BcpPubkeyQuery {
+  readonly publicKey: PublicKeyBundle;
```
Comment (webmaster128) [inline packages/iov-bcp-types/src/bcp.ts]: Can we try to use "pubkey" as a single work? This makes so much stuff simpler and shorter in the crypto world

File: packages/iov-bcp-types/src/bcp.ts [hunk 2]
```diff
@@ -91,13 +91,25 @@ export interface BcpValueNameQuery {
...
+}
+
+export type BcpAccountQuery = BcpAddressQuery | BcpPubkeyQuery | BcpValueNameQuery;
 
 // a type checker to use in the account-based queries
 export function isAddressQuery(query: BcpAccountQuery): query is BcpAddressQuery {
   return (query as BcpAddressQuery).address !== undefined;
 }
 
+export function isPublicKeyQuery(query: BcpAccountQuery): query is BcpPubkeyQuery {
```
Comment (webmaster128) [inline packages/iov-bcp-types/src/bcp.ts]: isPubkeyQuery

File: packages/iov-bcp-types/src/bcp.ts (hunk 1 above)
Comment (LucasIsasmendi) [reply packages/iov-bcp-types/src/bcp.ts]: We should unified this concept, because we are using multiple names in the same repo

Comment (webmaster128) [reply packages/iov-bcp-types/src/bcp.ts]: As discussed in Slack, we make "pubkey" an atomic word and use it at least in all new code and try to convert old code step by step

**Pull request merged**